specs/gloas/validator.md

```diff
@@ -162,12 +162,10 @@ construct the `payload_attestations` field in `BeaconBlockBody`:
   indices with respect to the PTC that is obtained from
   `get_ptc(state, block_slot - 1)`.
 
-#### Blob sidecars
+#### Constructing Data column sidecars
 
-The blob sidecars are no longer broadcast by the validator, and thus their
-construction is not necessary. This deprecates the corresponding sections from
-the Honest Validator specifications in the Fulu fork, moving them, albeit with
-some modifications, to the [Honest Builder](./builder.md) specifications.
+Unless a validator self-builds, the data column sidecars are no longer broadcast
+by the validator, and thus their construction is not necessary.
```

Comment:
> thus their construction is not necessary

So this is still inaccurate. If validators can/will self-build, their construction is necessary.

This sentence needs a complete re-write in my opinion.